lib/internal/util/inspect.js

```diff
@@ -2557,7 +2557,14 @@ function formatProperty(ctx, value, recurseTimes, key, type, desc,
         }
         ctx.indentationLvl -= 2;
       } catch (err) {
-        const message = `<Inspection threw (${err.message})>`;
+        let messageSuffix;
+        try {
+          // Error message itself may be a getter
+          messageSuffix = ` (${err.message})`;
+        } catch {
+          messageSuffix = '';
+        }
+        const message = `<Inspection threw${messageSuffix}>`;
```

Comment:
wdyt of doing this instead?
```suggestion
       let message = '<Inspection threw>';
       if (isErrorStackTraceLimitWritable()) {
         const stackTraceLimit = Error.stackTraceLimit;
         Error.stackTraceLimit = 0;
         try { message = `<Inspection threw (${inspect(err)})>`; }
         catch {}
         finally { Error.stackTraceLimit = stackTraceLimit; }
       } 
```

Reply:
This way the "Inspection threw" text is in two places, which seems strictly worse to me

Reply:
The suggestion is not about `messageSuffix`, it's about using `inspect`

Reply:
Can we first agree on the behavior? Any value that is thrown and has a `message` property which is a string will be displayed as the "inspection threw" detail.

I’ve currently implemented a `typeof message === "string"` check, should we also consider testing other types, or just display the value as-is (excluding `null` and `undefined`)?